src/resources/postcss/components/_navigation.pcss

```diff
@@ -0,0 +1,96 @@
+/* = Navigation
+============================ */
+.tribe-common {
+
+	.tribe-common-c-nav {
+		border-top: 1px solid var(--color-border-default);
+		padding-top: var(--spacer-6);
+
+		ul {
```

Comment:
could we add a class to the nav `ul` and `li` elements, such as `.tribe-common-c-nav__list` and `.tribe-common-c-nav__list-item` and use them as selectors rather than nesting `ul` and `li` under classes? this way we can keep the specificity of each element at a minimum.